testing/citest/spinnaker_testing/gcs_pubsub_trigger_agent.py

```diff
@@ -121,8 +121,14 @@ def detail(self):
 
   @property
   def timed_out(self):
+    ping = datetime.datetime.utcnow()
+    diff = ping - self.__start
+    return diff > datetime.timedelta(minutes=1)
+
+  def __check_finished_ok(self):
     resp = self.__trigger_response
-    return resp and resp.http_code == 504
+    executions = json.JSONDecoder().decode(resp.output) # Executions list
+    return resp.ok() and executions and len(executions) == 1
```

Comment:
executions wont be None (right?) so len(executions) == 1 is sufficient

Reply:
Done.